krypto/src/commonMain/kotlin/com/soywiz/krypto/encoding/Base64.kt

```diff
@@ -86,6 +98,10 @@ object Base64 {
 }
 
 fun String.fromBase64IgnoreSpaces(): ByteArray = Base64.decode(this.replace(" ", "").replace("\n", "").replace("\r", ""))
+fun String.fromBase64UrlIgnoreSpaces(): ByteArray = Base64.decode(this.replace(" ", "").replace("\n", "").replace("\r", ""), true)
```

Comment:
To be consistent: `fromBase64UrlIgnoreSpaces` -> `fromBase64UrlSafeIgnoreSpaces`

Reply:
Done